packages/frontend/app/components/ChangeLogModal.tsx

```diff
@@ -66,9 +66,9 @@ const ChangeLogModal: React.FC<ChangeLogModalProps> = ({
       classNames={{
         header: 'flex items-center justify-center',
       }}
-      className="box-border flex flex-col items-center justify-center overflow-y-auto"
+      className="box-border flex flex-col items-center justify-center"
     >
-      <div className="childrenMarkdownFormatted box-border h-[75vh] w-full overflow-y-auto px-4">
+      <div className="childrenMarkdownFormatted box-border h-[60vh] w-full overflow-y-auto px-4">
```

Comment:
I think it's easier to read if it takes up more of the page
```suggestion
      <div className="childrenMarkdownFormatted box-border h-[75vh] w-full overflow-y-auto px-4">
```

Reply:
I made this change because this:

![image](https://github.com/user-attachments/assets/a335fe36-7ae1-4805-9af5-d4a3df479b4e)

was how the mobile view looked. I feel like it being somewhat off the center didn't look great.

This is how the mobile view looks with 60vh.

![image](https://github.com/user-attachments/assets/819ebe20-b878-4dd6-8570-dd99c521be4e)

I agree that it looks better being centered in the space below the header bar tho for desktop so ill just add a media query.